events.h

```diff
@@ -1,20 +1,21 @@
-//
-// Created by Michael J Longfritz on 11/8/24.
-//
-
 #ifndef EVENTS_H
 #define EVENTS_H
 
-#include "modelStructures.h"
-
-enum EventType {
+typedef enum EventType {
   FERTILIZATION,
   HARVEST,
   IRRIGATION,
   PLANTING,
   TILLAGE,
   UNKNOWN_EVENT
-};
+} event_type_t;
+
+typedef enum IrrigationLocation
```

Comment:
in this context, the term 'location' is ambiguous - I was confused at first. The docs use the term 'type' but now I think 'method' would be better.

Reply:
With apologies for being pedantic, I went ahead and implemented the nomenclature change (location-->method) in #32 and will also update the docs.